client/src/lib/server/grpc.ts

```diff
@@ -25,44 +25,37 @@ export const proto = loadPackageDefinition(
     packageDefinition,
 ) as unknown as ProtoGrpcType;
 
-export const usersRustClient = new proto.proto.UsersService(
-    URI_USERS_RUST,
+const getCertificate = () =>
```

Comment:
Good idea, can i just ask for a small change? I am a type fo guy that use `const` for variables, and `function` for functions ;p

So just change the keyword? :)